v0/destinations/appcues/data/AppcuesPageProfile.json

```diff
@@ -0,0 +1,12 @@
+{
+	"userId": "userId",
```

Comment:
`userId` can come from multiple places. Also, is there any scope of sending `anonymousId` instead of `userId`?

Reply:
Yes it can come from multiple places. We couldn't send anonymousId instead of userId because the endpoint of Appcues needs the userId in its path